### ISSUE TITLE
Linux Manager:  'Add project' wizard text entry

### ISSUE THREAD

**Issue by RichardHaselgrove** [BOINC/boinc#5785]
**Describe the bug**
The text entry box for the user's email address on page 2 is not high enough - letters with descenders (g, q, y) are truncated and hard to proof-read.

Both versions are affected - new users creating an account, and returning members adding a new machine.

**System Information**
 - OS: Linux Mint 22
 - BOINC Version: 8.0.4

**Additional context**
This problem doesn't occur on Windows - the email address is fully legible. Please check for consequential changes in other OSs when making changes.

**Comment by Vulpine05** [comment]
I don't think this is a universal Linux issue.  I have check this on two separate computers with different versions of BOINC.

Below is a screenshot from a computer running BOINC 7.22.1 (both manager and client) with wxWidgets version 3.2.2 on Lubuntu 22.04:

![screen on 7 22 1](https://github.com/user-attachments/assets/08a321bc-4866-4083-ab71-fcc4c64420d5)

And here is a screenshot of Boinc 8.04 (again, both manager and client) with wxWidgets version 3.2.5 on Ubuntu 22.04.

![Screenshot from 2024-09-14 17-08-19](https://github.com/user-attachments/assets/4d24516d-6aa4-4900-88f5-ba7fefce1833)

I think the margin may be small, but to me the letter is clear.  @RichardHaselgrove, I assume you are seeing something different?

**Comment by RichardHaselgrove** [comment]
Well, this is what I see:
![image](https://github.com/user-attachments/assets/9436e4bb-8213-4727-9f54-941da2a677ba)
I used exactly the same email address as in your shots - but look what's happened to y, g, j.

Machine is brand new (30 August), running Linux Mint 22, running BOINC v8.0.4 from Berkeley install script, outputting to HD 1920 x 1200 screen.

**Comment by AenBleidd** [comment]
The differences you see are related to the GTK theme settings (and basically this is something you can tune via the Theme Manager in your OS settings).
Yes, that doesn't change that fact that this need to be fixed still, but it's not so easy when taking into the consideration the number of different Linux OS' and Desktop Managers' flavors.

**Comment by Vulpine05** [comment]
I figured that is what was happening to you, but I wanted to be sure.  Is this a BOINC problem or a Linux flavor problem?  I know we have preprocessor statements to distinguish between Linux, Windows, and Mac, but I haven't seen anything in the code that is special between Linux flavors.

I am not sure that something as universal as increasing the height of text boxes is an appropriate solution, it may create display problems in other ways.
